src/help/help_impl.hpp

```diff
@@ -366,49 +368,22 @@ bool is_visible_id(const std::string &id);
  */
 bool is_valid_id(const std::string &id);
 
-	// Helpers for making generation of topics easier.
+// Helpers for making generation of topics easier.
 
 inline std::string make_link(const std::string& text, const std::string& dst)
-	{
-		// some sorting done on list of links may rely on the fact that text is first
-		return "<ref>text='" + help::escape(text) + "' dst='" + help::escape(dst) + "'</ref>";
-	}
-
-inline std::string jump_to(const unsigned pos)
-	{
-		std::stringstream ss;
-		ss << "<jump>to=" << pos << "</jump>";
-		return ss.str();
-	}
-
-inline std::string jump(const unsigned amount)
-	{
-		std::stringstream ss;
-		ss << "<jump>amount=" << amount << "</jump>";
-		return ss.str();
-	}
+{
+	// some sorting done on list of links may rely on the fact that text is first
```

Comment:
This comment is now false – the text is not first anymore. Therefore, I think you need to investigate if there is indeed sorting of links that relies on the text being first, and if so fix it; and also remove the comment.